api/src/controllers/utils.ts

```diff
@@ -124,13 +122,39 @@ router.post(
 		const busboy = Busboy({ headers });
 
 		busboy.on('file', async (_fieldname, fileStream, { mimeType }) => {
-			try {
-				await service.import(req.params['collection']!, mimeType, fileStream);
-			} catch (err: any) {
-				return next(err);
-			}
-
-			return res.status(200).end();
+			const { tmp } = await import('@directus/utils/node');
```

Comment:
Use dynamic import to reduce the time required on server to start, since it's not required elsewhere.